<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBNetEditorDialog</class>
 <widget class="QDialog" name="RBNetEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>&amp;Пол</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>221</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbBegAgeUnit">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>&amp;Возраст</string>
     </property>
     <property name="buddy">
      <cstring>cmbBegAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="edtBegAgeCount">
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>321</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="edtEndAgeCount">
     <property name="inputMask">
      <string>000; </string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lblAgeSep">
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>cmbEndAgeUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="cmbEndAgeUnit">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="0" column="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>321</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>На&amp;именование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="edtName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QCheckBox" name="chkUseInCreateClients">
     <property name="text">
      <string>Применять ограничения при регистрации пациента</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBNetEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBNetEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
